packages/odyssey-react-mui/src/NativeSelect.tsx

```diff
@@ -98,13 +110,15 @@ const NativeSelect: ForwardRefWithType = forwardRef(
     HasMultipleChoices extends boolean
   >(
     {
+      autoCompleteType,
       defaultValue,
       errorMessage,
       errorMessageList,
       hasMultipleChoices: hasMultipleChoicesProp,
       hint,
       HintLinkComponent,
       id: idOverride,
+      inputFocusRef,
```

Comment:
I'd like this to be more generic.

Can we rename this `inputRef` so maybe `.blur()` is possible in the future and others?

The internal one could be named `localInputRef` to differentiate it.